src/content/docs/tutorial-service-level-mgmt/establish-output-sli.mdx

```diff
@@ -0,0 +1,100 @@
+* Located your application in our [entity explorer](/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts#find).
+
+In the entity explorer, find your application (entity type of `Services - APM`) and select it. You should see the overview screen below. **Don't click `Service levels` yet.**
+
+<img
+  alt="APM Overview screenshot"
+  src={apmOverviewScreen}
+/>        
+    </Step>
+    <Step>
+## Identify your service boundary [#identifying-service-boundary]
+
+Next, you'll want to ensure you're measuring the output of your service. While the dependencies of an application each play a part in response times and success rates, you can measure the final and total response times and success at the point where the request is received and responded to.
+
+You can use [service maps](/docs/new-relic-one/ui-data/service-maps/how-use-service-maps/) and [automaps](/docs/new-relic-one/ui-data/automaps) to help determine if the application you're looking at is a dependency or runs the endpoint API.
+
+In the screenshot example below, you can see all applications that support order processing. We start with selecting #2 (Order-Composer). Then, we click **Service maps** and discover that **Order-Composer** is really a dependency. You'll need to select #1 (Order-Processing) in order to establish a true health service level.
+
+<img
+  alt="Service map example"
+  src={apmServiceBoundary}
+/>        
+
+<Callout variant="tip">
+Your team may only be accountable for the **Order-Composer** dependency. If so, then your own service level on **Order-Composer** can self-monitor the performance. Ensure you tag your own non-customer facing service levels as `customer-facing:false` to allow for better filtering in health reports. Also, consider working with the customer-facing endpoint (#1 Order-Processing) during observability to create true output performance, an input connectivity service level, and client service levels.
+</Callout>
+
+    </Step>
+    <Step>
+## Create your baseline
+
+You must also establish a baseline to implementat service levels. Creating a baseline allows you to measure the current performance of a service, then use service level reports to find out if you're hitting that baseline or degrading. You can create a baseline for almost any dataset; however, you use different formulas and recommendations in different use cases.
+
+When you create service levels, you should start with the output performance of your applications. You can use response times (latency) and percentage of non-errors (success). When you do this, you create a reliability health metric. Seasonality and peak usage doesn't handicap your performance, and the more history you include in your measurement, the more likely you're including different codebases from releases. Previous deployments, no matter how small, could skew your results.
```

Comment:
OK -- So once you've determined the "thing" you should create a service level for (with the service map), you then need to "find the baseline." The baseline helps you make a decision about what your SL should be

Reply:
I reworked this section a bit, moved some things around and added some clarification, so these issues SHOULD be fixed now